src/window.rs

```diff
@@ -953,6 +968,16 @@ impl Window {
         self.window.theme()
     }
 
+    /// Prevents the window contents from being captured by other apps.
+    ///
```

Comment:
> > This is OS API limitation. We use `[NSWindow setSharingType:NSWindowSharingNone]`, which does not prevent QuickTime from capturing the window unfortunately.

Fair enough - would you mind documenting that here as well?

Reply:
sure